R/.Rprofile

```diff
@@ -1,3 +1,69 @@
+# Source VSCode-R options
```

Comment:
`.Rprofile` only applies to session started from `R: Create R terminal`. User managed sessions do not respect this profile. Maybe we need to put it back to `init.R` so that in both cases the settings are used.

Reply:
Thanks, I wasn't aware of this - I've added it back and put in a check for already-set options